pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/ReplicatedSubscriptionsController.java

```diff
@@ -206,6 +206,12 @@ private void receiveSubscriptionUpdated(ReplicatedSubscriptionsUpdate update) {
     private void startNewSnapshot() {
```

Comment:
I think the better way is not to create `ReplicatedSubscriptionsController` when turning off `replication.`